readme.md

```diff
@@ -246,7 +246,44 @@ public class Main {
 }
 ```
 
-Additional options are provided for adding environment variables from fields, config maps and secrets.    
+Additional options are provided for adding environment variables from fields, config maps and secrets. 
+
+#### Adding environmnet variables from ConfigMap
+
+To add an environment variable that points to a ConfigMap property, you need to specify the configmap using the `configmap` property in the annotation.
```

Comment:
I would change  `in the annotation` --> `in the @Env annotation` (My first reflection has been to look for an `@ConfigMap` annotation in the code to verify this field)

Reply:
fixed.